### PR TITLE
Add weight_threshold in the authorities tab

### PR DESCRIPTION
**Pull request by freak5g**
#35

Screenshot from 2023-05-17 16-40-09
Screenshot from 2023-05-17 16-40-50

### CODE REVIEW

File: app/sections/assets/asset.html [hunk 1]
```diff
@@ -95,7 +95,7 @@ <h2>{{data.symbol}}</h2>
                         <span data-translate="Description"></span>
                     </li>
                     <li class="list-group-item">
-                        <span class="badge" style="background-color: #FFFFFF; color: #000000">{{data.max_supply | number}} {{data.symbol}}</span>
+                        <span class="badge" style="background-color: #FFFFFF; color: #000000">{{data.max_supply}} {{data.symbol}}</span>
```
Comment (sschiessl-bcp) [inline app/sections/assets/asset.html:98]: Why did you remove the number formatting?

Comment (freak5g) [reply app/sections/assets/asset.html:98]: "If the input is null or undefined, it will just be returned. If the input is infinite (Infinity or -Infinity), the Infinity symbol '∞' or '-∞' is returned, respectively. If the input is not a number an empty string is returned."
Because the input is already formatted. An alternative approach would be to remove any prior formatting, not sure what's optimal.

Comment (freak5g) [reply app/sections/assets/asset.html:98]: https://github.com/blocksights/blocksights-open-explorer/blob/a5fbc22a8173fff0deac9130a72d4ecbe1ae61b2/app/core/app.utilities.js#L14

**Comment by abitmore** [comment]
It seems that this PR has not been deployed yet?

**Comment by sschiessl-bcp** [comment]
The cronjob for fetching the binaries was broken due to github ip change, thanks! works again now

**Comment by abitmore** [comment]
It seems the change is still not live on blocksights.info.
image

**Comment by sschiessl-bcp** [comment]
It does work for committee-account 
image

**Comment by abitmore** [comment]
It seems https://github.com/blocksights/blocksights-open-explorer/commit/8c4045023d54379c6d4c22466e4a65d512949c97 should fix it. Need to manually deploy again?
